src/Login.php

```diff
@@ -67,4 +69,108 @@ public function isSponsoring(string $account): bool
         return ($result['user']['isSponsoredBy'] ?? false) ||
             ($result['organization']['isSponsoredBy'] ?? false);
     }
+
+    public function hasSponsors(): bool
+    {
+        $query = <<<'QUERY'
+            query (
+                $account: String!
+            ) {
+                user(login: $account) {
+                    sponsorshipsAsMaintainer {
+                        totalCount
+                    }
+                }
+                organization(login: $account) {
+                    sponsorshipsAsMaintainer {
+                        totalCount
+                    }
+                }
+            }
+        QUERY;
+
+        $result = $this->client->send($query, [
+            'account' => $this->login,
+        ]);
+
+        return ($result['user']['sponsorshipsAsMaintainer']['totalCount'] ?? $result['organization']['sponsorshipsAsMaintainer']['totalCount']) > 0;
+    }
+
+    public function sponsors(array $select = ['login']): LazyCollection
+    {
+        $fields = implode(PHP_EOL, $select);
```

Comment:
Since `login` is mandatory for running the query I think it's best that we always require it.

```suggestion
    public function sponsors(array $select = []): LazyCollection
    {
        $fields = implode(PHP_EOL, array_unique($select + ['login']));
```

Reply:
It's not - You have to select at least one field - but `login` doesn't have to be selected. You can just select `databaseId` or any other field if you don't need the login name.

Reply:
Ah okay gotcha. In that case can we maybe require an empty array for the method signature and use the login field as a fallback when nothing is passed?

Reply:
It's just that I think an empty array is a cleaner default here.

Reply:
In that case I would remove the default at all and require the user to always pass in the fields.
If the default is in the signature it's immediately visible to the user. If it's somewhere in the method body it's just some invisible magic.

Reply:
You're right about that. That's a good remark. Let's leave the default then as is 👍